README.md

```diff
@@ -58,21 +58,26 @@ Before running the script, you'll make changes specific to your environment.
 In `cluster_configuration/cluster-startup.sh`:
 
     - Update `TPCX_BB_HOME=...` to location on disk of this repo
-    - Update `INTERFACE=...` to refer to the relevant network interface present on your cluster.
     - Update `CONDA_ENV_PATH=...` to refer to your conda environment path.
     - Update `CONDA_ENV_NAME=...` to refer to the name of the conda environment you created, perhaps using the `yml` files provided in this repository.
-    - Update `SCHEDULER=...` to refer to the host name of the node you intend to use as the scheduler.
-    - Update `SCHEDULER_FILE=...` to refer to the location of your scheduler file
+    - Update `CLUSTER_MODE="NVLINK"` to refer to your communication method, either "TCP" or "NVLINK".
+    - You may also need to change the `LOCAL_DIRECTORY` and `WORKER_DIR` depending on your filesystem. Make sure that these point to a location to which you have write access and that `LOCAL_DIRECTORY` is accessible from all nodes.
 
-In `cluster_configuration/example-cluster-scheduler.json`:
-Update the scheduler address to be the address for the network interface you chose for `INTERFACE=...` above. If you are not using UCX, you'll need to adjust the address to be `tcp://...` rather than `ucx://...`. Note that `example-cluster-scheduler.json` is just an example scheduler configuration. See [the Dask docs](https://docs.dask.org/en/latest/setup/hpc.html#using-a-shared-network-file-system-and-a-job-scheduler) for more details on how you can generate your own and make it available to all cluster nodes.
 
-To start up the cluster, please run the following on every node from `tpcx_bb/cluster_configuration/`.
+To start up the cluster on your scheduler node, please run the following from `tpcx_bb/cluster_configuration/`.
```

Comment:
```suggestion
To start up the cluster on your scheduler node, please run the following from `tpcx_bb/cluster_configuration/`. This will spin up a scheduler and one Dask worker per GPU.
```